raster-foundry/raster-foundry
# Allow organization members to create teams if they aren't an admin.

## Overview
Automatically add them to the team as a team admin so they can manage it if they
are not an organization admin

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~

## Testing Instructions

 * As a non-admin in an organization, create a team and verify that you are placed in it automatically
* As an admin of an organization, verify that creating a team does not place you inside it automatically.

Closes #4135
